### ISSUE TITLE
How to modify the validation set and test set?

### ISSUE THREAD

**Issue by Stangerine** [LitingLin/LoRAT#5]
How to modify the validation set and test set? I modified the training set, but the validation set and test set are still got-10k-val and got-10k-test
![image](https://github.com/user-attachments/assets/ea3a28b0-cbe4-4b53-8ff3-2d7be26422ca)

**Comment by Sherlock0417** [comment]
I want to know if you solved the problem? I had a similar problem

**Comment by Situ-Weixi** [comment]
I want to know if you have solved the problem? I had a similar problem.
![2024-12-11_20-50](https://github.com/user-attachments/assets/c78ad6ba-c020-4d18-a030-b13f2fed4a2e)
